api/controllers/game/play-again.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Indicate that the player wants to play again
+ */
+const gameAPI = sails.hooks['customgamehook'];
+module.exports = async function (req, res) {
+  try {
+    const { game: gameId, pNum } = req.session;
+    const game = await Game.findOne({ id: gameId }).populate('players');
+    const gameUpdates = {};
+    if (pNum === 0 || pNum === '0') {
+      gameUpdates.p0Rematch = true;
+    } else {
+      gameUpdates.p1Rematch = true;
+    }
```

Comment:
```suggestion
    const gameUpdates = { [`p${pNum}Rematch`]:true };

```

I'm not sure if this is to complicated for our sake, @itsalaidbacklife  what do you think?